packages/app/src/components/pw-manager-input.tsx

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
 
-export const usernameInputId = 'username';
+interface PasswordManagerHiddenInput {
+  secretKey?: string;
+  appName?: string;
+}
 
-export const PasswordManagerHiddenInput = ({ secretKey }: { secretKey?: string }) => {
+const visuallyHiddenProps = { opacity: 0, position: 'absolute' } as const;
+
+const passwordManagerUsername = (appName?: string) => 'Blockstack Secret Key' + (!!appName && ` ${appName} `);
+
+export const PasswordManagerHiddenInput = ({ secretKey, appName }: PasswordManagerHiddenInput) => {
   if (!secretKey) return null;
   return (
     <>
-      <label style={{ opacity: 0, position: 'absolute' }} aria-hidden htmlFor={usernameInputId}>
-        Username
-      </label>
-      <label style={{ opacity: 0, position: 'absolute' }} aria-hidden>
-        Secret Key
+      <label style={visuallyHiddenProps} htmlFor="username">
         <input
-          id="secret-key"
-          type="password"
-          autoComplete="new-password"
-          defaultValue={secretKey}
-          aria-hidden="true"
+          id="username"
+          name="username"
+          type="text"
+          autoComplete="username"
+          value={passwordManagerUsername(appName)}
```

Comment:
I'm thinking that the app name might not be the best thing for "username" here. I think something hard-coded like "Blockstack Secret Key" would be better than the app name.

Or, do you know if it's possible to have this work with the username being blank?

Reply:
This will save as `Blockstack Secret Key (<appName>)`. Do we want something else?

👎 to blank idea, this will create problems

Reply:
Yeah, ok, I think this solution is good then. Although, the function doesn't put parentheses around `appName` - I think that would be better.